pkg/hook/hook_manager.go

```diff
@@ -103,6 +103,10 @@ func (hm *hookManager) Init() error {
 	hm.hooksInOrder = make(map[BindingType][]*Hook)
 	hm.hooksByName = make(map[string]*Hook)
 
+	if err := utils_file.RecursiveCheckLibDirectory(hm.workingDir); err != nil {
```

Comment:
We can run both functions in separate goroutines to cut the time in half.

This will be especially noticeable on projects with a large number of files in the hooks directory.

But I don't know how common this practice is in the company. Is it worth spending time on performance?

Reply:
It's a one-time operation, so we don't need this optimization now.